release/edgex-docs.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: 'edgex-docs'
+version: '3.1.0'
```

Comment:
Is this the version that the `edgexversion` maco will be set to? If so this needs to be 3.1

Reply:
Not sure, for Minnesota we had `3.0.0` for this: https://github.com/edgexfoundry/cd-management/blob/release/release/edgex-docs.yaml

@ernestojeda, what do you think?

Reply:
Yes, but I we automatically truncate it to the minor version so, 3.1.0 would become 3.1